elixir/apps/domain/lib/domain/actors.ex

```diff
@@ -363,6 +363,15 @@ defmodule Domain.Actors do
     end
   end
 
+  def list_actors_by_type(%Auth.Subject{} = subject, type, opts \\ []) do
```

Comment:
This can be replaced with a `type` filter in `list_actors/3` function.

Reply:
👍  This has been updated.